data/mods/Magiclysm/Spells/attunements/Shapeshifter.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "type": "SPELL",
+    "id": "shapeshifter_alkahest",
+    "name": "Alkahest",
+    "description": "Release the mana-saturated liquid out your ley lines and cover everything with bioactive solvent.  Sadly you are the only person that have resistance to his effect, so its better to be alone when you cast it.",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "effect": "attack",
+    "shape": "cone",
+    "damage_type": "acid",
+    "max_level": 25,
```

Comment:
attunement spells are supposed to be balanced around max level 35

Reply:
whoops, my bad